tensorflow/examples/image_retraining/retrain.py

```diff
@@ -450,7 +452,14 @@ def get_or_create_bottleneck(sess, image_lists, label_name, index, image_dir,
                                 category)
     if not gfile.Exists(image_path):
       tf.logging.fatal('File does not exist %s', image_path)
-    image_data = gfile.FastGFile(image_path, 'rb').read()
+    jpgext = ['jpg', 'jpeg', 'JPG', 'JPEG']
+    if (image_path.split('.')[-1] in jpgext):
```

Comment:
os.path.splitext()  returns '.jpg', so it has to be set [1:] to return 'jpg' like:
`os.path.splitext(image_path)[1][1:]` 
I think `image_path.split('.')[-1]` might be  clear enough.